src/core/api.md

```diff
@@ -4,6 +4,11 @@
 
 ```ts
 
+import type { CorePlatform as CorePlatform_2 } from '@builder.io/qwik';

```

Comment:
Not a blocker, but why is this happening? `CorePlatform as CorePlatform_2`

Reply:
I saw it too, I think its a bundling of types, going to dig deeper.